packages/dd-trace/test/appsec/waf/waf_context_wrapper.spec.js

```diff
@@ -27,14 +41,45 @@ describe('WAFContextWrapper', () => {
     wafContextWrapper.run(payload)
 
     expect(ddwafContext.run).to.have.been.calledOnceWithExactly(payload, 1000)
+    expect(reportMetricsStub).to.have.been.calledOnce
   })
 
-  it('Should send ephemeral addresses every time', () => {
+  it('Should send HTTP_INCOMING_QUERY twice if waf run returns null', () => {
     const ddwafContext = {
       run: sinon.stub()
```

Comment:
to make it clear that it's not because it returns undefined, but because it throws
```suggestion
      run: sinon.stub().throws(new Error('test'))
```